library/classes/Installer.class.php

```diff
@@ -19,41 +19,41 @@
 
 class Installer
 {
-    private $iuser;
-    private $iuserpass;
-    private $iuname;
-    private $iufname;
-    private $igroup;
-    private $i2faEnable;
-    private $i2faSecret;
-    private $server;
-    private $loginhost;
-    private $port;
-    private $root;
-    private $rootpass;
-    private $login;
-    private $pass;
-    private $dbname;
-    private $collate;
-    private $site;
-    private $source_site_id;
-    private $clone_database;
-    private $no_root_db_access;
-    private $development_translations;
-    private $new_theme;
-    private $ippf_specific;
+    public $iuser;
```

Comment:
I'm not fond of public class variables as it makes it difficult to refactor, but since this is just to fix existing installs I suppose its whats required.